cmd/up.go

```diff
@@ -102,3 +122,43 @@ var up = &cobra.Command{
 		}
 	},
 }
+
+func writePidFile() error {
+	if pidFile == "" {
+		return nil
+	}
+
+	if err := os.MkdirAll(filepath.Dir(pidFile), 0o755); err != nil {
+		return err
+	}
+	if stats, err := os.Stat(pidFile); err == nil {
+		if stats.Size() > 0 {
+			return errors.New("pidFile already exists")
```

Comment:
How about adding the path to `pidFile` in this error output?
```suggestion
			return errors.Newf("pidFile %q already exists", pidFile)
```